int32 track_id
float32 score
float32 bbox_center_x
float32 bbox_center_y
float32 bbox_width
float32 bbox_height
HorseKeypoint[] keypoints
